group/definitions.js

```diff
@@ -0,0 +1,96 @@
+const messageId = {
+  type: 'string',
+  $comment: '42 chars + 1 char from a limited subset (last bits are 0) + 1 =',
+  pattern: '^ssb:message/classic/[a-zA-Z0-9_\\-]{42}[AEIMQUYcgkosw048]=$'
+}
+
+const groupId = {
+  type: 'string',
+  pattern: '^ssb:identity/group/[a-zA-Z0-9_\\-]{42}[AEIMQUYcgkosw048]=$'
+}
+
+const feedId = {
+  type: 'string',
+  pattern: '^ssb:feed/classic/[a-zA-Z0-9_\\-]{42}[AEIMQUYcgkosw048]=$'
```

Comment:
:fire: this is gonna bite us in the ass when someone wants to make a buttwooo based feed

```suggestion
  pattern: `^ssb:feed/(${validFeeds.join('|')})/[a-zA-Z0-9_\\-]{42}[AEIMQUYcgkosw048]=$`
```

we could do something like this... but maybe it's another PR later

Reply:
yeah i think that's for later